performance/SwiftUIPerformanceQuickstart/PerformanceExample/Podfile

```diff
@@ -0,0 +1,34 @@
+# Uncomment the next line to define a global platform for your project
+
+target 'PerformanceExample (iOS)' do
+  platform :ios, '14.0'
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for PerformanceExample (iOS)
+  pod 'Firebase'
+  pod 'FirebasePerformance'
+
+end
+
+target 'Tests_iOS' do
+  platform :ios, '14.0'
```

Comment:
Is 13.0 feasible?

Reply:
Some SwiftUI features are not available and would break the app.